CMake/cdat_modules/vtk_external.cmake

```diff
@@ -73,6 +73,12 @@ if(CDAT_BUILD_OFFSCREEN)
   )
 endif()
 
+if(CDAT_BUILD_VTKWEB)
```

Comment:
`CDAT_BUILD_*` is consistent with the other options.  To me, `VTKWEB` is more explicit unless there is something else that `WEB` will enable, but I don't have a strong opinion.